consensus/broadcast/metaChainMessenger.go

```diff
@@ -160,6 +161,7 @@ func (mcm *metaChainMessenger) PrepareBroadcastHeaderValidator(
 	err = mcm.delayedBlockBroadcaster.SetHeaderForValidator(vData)
 	if err != nil {
 		log.Error("metaChainMessenger.PrepareBroadcastHeaderValidator", "error", err)
+		return
```

Comment:
this return is redundant

Reply:
I would keep it just for the case when the method would be extended by someone who would forgot to added back